src/components/EditBook/EditBookAction.jsx

```diff
@@ -0,0 +1,14 @@
+const EditBookAction = () => {
+  return (
+    <div className="edit-book-action_container">
+      <button type="button" className="edit-book-action_borrowed">
+        Mark as borrowed
+      </button>
+      <button type="button-delete" className="edit-book-action_delete">
```

Comment:
`type` is a built-in attribute that is used by the browser to understand what the button does and you can't create your own types. It will have an action on click so `type="button"` is good in that case as well

```suggestion
      <button type="button" className="edit-book-action_delete">
```